docs/how-prysm-works/overview-technical.md

```diff
@@ -11,11 +11,11 @@ sidebar_label: Architecture overview
 
 When a Prysm client is initialised out of the box, it starts a variety of services that run in parallel to handle everything required for the lifecycle of the beacon chain. In no particular order, Prysm includes:
 
-* A [**beacon node**](./beacon-node) which powers the beacon chain at the core of Ethereum 2.0.
-* A [**validator client**](./validator-clients) connects to the beacon node and manages staking keypairs.
-* A [**public RPC server**](./ethereum-2-public-api) to request data about network state, blocks, validators etc.
-* A [**persistent key-value store**](./database-backend-boltdb) in the form of a database \([BoltDB](./database-backend-boltdb)\).
-* A [**P2P networking framework and server**](./p2p-networking) to connect with other beacon nodes.
+* A [**beacon node**](../beacon-node) which powers the beacon chain at the core of Ethereum 2.0.
```

Comment:
Use full paths for everything please we can't guarantee the future structure of the docs and it becomes a real mess to have to go back and fix them after the fact (as seen by these being broken in the first place)